modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -592,6 +592,8 @@ export class Wallet implements IWallet {
    *                  - maximum number of unspents you want to use in the transaction
    * Output parameters:
    * @param {Number} params.numUnspentsToMake - the number of new unspents to make
+   * @param {Boolean} params.bulk - if set to True, this enables the consolidation of large number of unspents by creating multiple transactions,
+   *                                with each transaction composed of 200 unspents, except for the last transaction which may have fewer unspents.
    */
   private async manageUnspents(
```

Comment:
can we improve the return type here? 

how do the callers handle the array response?

Reply:
It will be the return type of send API.

From `iWallet.ts`

```
send(params?: SendOptions): Promise<any>;
sendMany(params?: SendManyOptions): Promise<any>;
```